contracts/CurrencyNetwork.sol

```diff
@@ -755,17 +728,14 @@ contract CurrencyNetwork is CurrencyNetworkInterface, Authorizable, debtTracking
         bool reducingDebtOnly = true;
 
         // check path starting from sender correctly accumulate the fee
-        for (uint i = 0; i < _path.length; i++) {
-            // the address of the receiver is _path[i]
-            address sender;
+        for (uint i = 0; i < _path.length-1; i++) {
```

Comment:
rename i to sender_index?
Or shift index to use receiver index as in the other function?
Pro/Cons?

Reply:
I renamed it to senderIndex. I don't like shifting it because it feels natural for this type of transfer to think from sender to receiver while it is natural in the other to think from receiver to sender.